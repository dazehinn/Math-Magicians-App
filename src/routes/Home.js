import styles from './Home.module.css';

const Home = () => (
  <div className={styles.main}>
    <h2>Welcome to our page</h2>
    <p>
      Math Magicians is an app for all lovers of mathematics. It is build with React and much love.
      As a user, you are given access to a simple calcular and quotes from famous persons about
      life. Feel free to explore this app and send your impressions by &nbsp;
      <a href style={{ color: 'lightblue', textDecoration: 'underline' }}>clicking here</a>
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi quaerat eius quod ducimus iusto
      sit impedit etnam quos aliquam maiores expedita, similique libero voluptate nihil hic mollitia
      maxime modi soluta officia cumque error. Pariatur libero unde nobis cupiditate voluptatum
      ipsa itaque natus rem culpa.
    </p>
  </div>
);

export default Home;
