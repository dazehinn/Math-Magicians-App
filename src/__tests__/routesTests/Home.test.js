import renderer from 'react-test-renderer';
import Home from '../../routes/Home';

describe('Home Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
