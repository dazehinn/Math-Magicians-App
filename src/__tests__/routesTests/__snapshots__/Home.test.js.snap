// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Tree 1`] = `
<div
  className="main"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Math Magicians is an app for all lovers of mathematics. It is build with React and much love. As a user, you are given access to a simple calcular and quotes from famous persons about life. Feel free to explore this app and send your impressions by Â 
    <a
      href={true}
      style={
        Object {
          "color": "lightblue",
          "textDecoration": "underline",
        }
      }
    >
      clicking here
    </a>
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi quaerat eius quod ducimus iusto sit impedit etnam quos aliquam maiores expedita, similique libero voluptate nihil hic mollitia maxime modi soluta officia cumque error. Pariatur libero unde nobis cupiditate voluptatum ipsa itaque natus rem culpa.
  </p>
</div>
`;
